### ISSUE TITLE
[Bug]: Edit locally - Could not find a file for local editing

### ISSUE THREAD

**Issue by vagner-dias** [nextcloud/desktop#5341]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

When Using the Edit Locally feature on NC25.0.2 + Windows Client 3.6.5, an error is triggered as shown below:

The same happens to any file I try to edit.

![2023-01-19_09h56_53](https://user-images.githubusercontent.com/86367839/213454885-b827f7d4-40ef-4b1c-8c7a-6f6b079ed930.png)

### Steps to reproduce

Scenario:
- OS: Windows 10
- NextCloud client is already installed.
- Browser: Chrome (latest version)

Browse for any file, on any folder, 
Click on the ellipsis, 
Choose Edit Locally
Allow chrome to trigger NextCloud Client

The error is shown as above

### Expected behavior

The file should open

### Which files are affected by this bug

any file

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10

### Package

Appimage

### Nextcloud Server version

25.0.2

### Nextcloud Desktop Client version

3.6.5

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
Client logs:

2023-01-19 10:31:58:310 [ info nextcloud.gui.application C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\application.cpp:551 ]:	################## "Nextcloud" locale: "pt_BR" ui_lang: QVariant(QString, "en_US") version: "3.6.5stable-Win64 (build 20230118)" os: "Windows 10 Version 2009"
2023-01-19 10:31:58:311 [ info nextcloud.gui.application C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\application.cpp:556 ]:	Arguments: ("C:\\Program Files\\Nextcloud\\nextcloud.exe")
2023-01-19 10:31:58:311 [ debug nextcloud.gui.systray C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\systray.cpp:293 ]	[ OCC::Systray::createEditFileLocallyLoadingDialog ]:	Opening a file local editing dialog...
2023-01-19 10:31:58:324 [ info nextcloud.sync.accessmanager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\accessmanager.cpp:78 ]:	4 "" "https://<<HIDDEN>>/ocs/v2.php/apps/files/api/v1/openlocaleditor/tHB5cjE9a1HLS1V0ZaXoazsWGpF7lgeehxJNAgU2UIITcuCZS2KptSuhsWIaH9eNnvHNAiIMU4p7wet6J4Ob2uqY84vzw5GfupWHT69VkpVkMmrPQ5WXpGPCyf0fXBN3?path=/BitLocker Recovery Key 6A678353-4EFB-4387-AAA9-B258A2B21917.TXT" has X-Request-ID "da04d267-d284-492a-9fed-48f565dd18ec"
2023-01-19 10:31:58:324 [ debug nextcloud.sync.cookiejar C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\cookiejar.cpp:90 ]	[ OCC::CookieJar::cookiesForUrl ]:	QUrl("https://<<HIDDEN>>/ocs/v2.php/apps/files/api/v1/openlocaleditor/tHB5cjE9a1HLS1V0ZaXoazsWGpF7lgeehxJNAgU2UIITcuCZS2KptSuhsWIaH9eNnvHNAiIMU4p7wet6J4Ob2uqY84vzw5GfupWHT69VkpVkMmrPQ5WXpGPCyf0fXBN3?path=/BitLocker Recovery Key 6A678353-4EFB-4387-AAA9-B258A2B21917.TXT") requests: (QNetworkCookie("oc_sessionPassphrase=kKWOf4gzXCVj%2BcZAZ%2B33YflPUD1MSSF6IduDFqnlFsq7edxySzFIgenM9UFao3G7bPGfmi2kMgy0uxBBsCJ5A7cDbRh4m1JfqDGuSSxxRZT4SQuop%2F7zSq1zEy%2Bq0goK; secure; HttpOnly; domain=<<HIDDEN>>; path=/"), QNetworkCookie("__Host-nc_sameSiteCookielax=true; secure; HttpOnly; expires=Fri, 31-Dec-2100 23:59:59 GMT; domain=<<HIDDEN>>; path=/"), QNetworkCookie("__Host-nc_sameSiteCookiestrict=true; secure; HttpOnly; expires=Fri, 31-Dec-2100 23:59:59 GMT; domain=<<HIDDEN>>; path=/"), QNetworkCookie("ocsfd6u2aylg=db15213a54441d70fde831633943a3d2; secure; HttpOnly; domain=<<HIDDEN>>; path=/"))
2023-01-19 10:31:58:325 [ info nextcloud.sync.networkjob C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\abstractnetworkjob.cpp:361 ]:	OCC::SimpleApiJob created for "https://<<HIDDEN>>" + "/ocs/v2.php/apps/files/api/v1/openlocaleditor/tHB5cjE9a1HLS1V0ZaXoazsWGpF7lgeehxJNAgU2UIITcuCZS2KptSuhsWIaH9eNnvHNAiIMU4p7wet6J4Ob2uqY84vzw5GfupWHT69VkpVkMmrPQ5WXpGPCyf0fXBN3" ""
2023-01-19 10:31:58:325 [ warning default unknown:0 ]:	content-type missing in HTTP POST, defaulting to application/x-www-form-urlencoded. Use QNetworkRequest::setHeader() to fix this problem.
2023-01-19 10:31:58:893 [ info nextcloud.sync.credentials.webflow C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\creds\webflowcredentials.cpp:426 ]:	request finished
2023-01-19 10:31:58:894 [ debug nextcloud.sync.networkjob.simpleapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\networkjobs.cpp:1178 ]	[ OCC::SimpleApiJob::finished ]:	result:  "/ocs/v2.php/apps/files/api/v1/openlocaleditor/tHB5cjE9a1HLS1V0ZaXoazsWGpF7lgeehxJNAgU2UIITcuCZS2KptSuhsWIaH9eNnvHNAiIMU4p7wet6J4Ob2uqY84vzw5GfupWHT69VkpVkMmrPQ5WXpGPCyf0fXBN3" "Unknown error" 200
2023-01-19 10:31:58:894 [ info nextcloud.gui.editlocallyjob C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\editlocallyjob.cpp:112 ]:	token check result 200
2023-01-19 10:31:58:894 [ debug nextcloud.gui.systray C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\systray.cpp:310 ]	[ OCC::Systray::destroyEditFileLocallyLoadingDialog ]:	Closing a file local editing dialog...
2023-01-19 10:31:58:894 [ warning nextcloud.gui.activity C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\tray\usermodel.cpp:530 ]:	Item  "Nextcloud"  retrieved resulted in  "Could not find a file for local editing. Make sure it is not excluded via selective sync."
2023-01-19 10:31:58:894 [ info nextcloud.gui.activity C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\tray\activitylistmodel.cpp:552 ]:	Error successfully added to the notification list:  "BitLocker Recovery Key 6A678353-4EFB-4387-AAA9-B258A2B21917.TXT"
2023-01-19 10:31:58:921 [ debug com.nextcloud.owncloudgui C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\owncloudgui.cpp:375 ]	[ OCC::ownCloudGui::slotShowTrayMessage ]:	Going to show notification with title: ' "BitLocker Recovery Key 6A678353-4EFB-4387-AAA9-B258A2B21917.TXT" ' and message: ' "Could not find a file for local editing. Make sure it is not excluded via selective sync." '
2023-01-19 10:31:58:932 [ debug nextcloud.sync.networkjob C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\abstractnetworkjob.cpp:298 ]	[ OCC::AbstractNetworkJob::slotFinished ]:	Network job OCC::SimpleApiJob finished for "/ocs/v2.php/apps/files/api/v1/openlocaleditor/tHB5cjE9a1HLS1V0ZaXoazsWGpF7lgeehxJNAgU2UIITcuCZS2KptSuhsWIaH9eNnvHNAiIMU4p7wet6J4Ob2uqY84vzw5GfupWHT69VkpVkMmrPQ5WXpGPCyf0fXBN3"
2023-01-19 10:32:01:681 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:896 ]:	Etag poll timer timeout
2023-01-19 10:32:01:681 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:900 ]:	Folders to sync: 1
2023-01-19 10:32:01:681 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:910 ]:	Number of folders that don't use push notifications: 0
2023-01-19 10:32:23:799 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:257 ]	[ OCC::PushNotifications::pingWebSocketServer ]:	Ping websocket server
2023-01-19 10:32:24:006 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:237 ]	[ OCC::PushNotifications::onWebSocketPongReceived ]:	Pong received in time
2023-01-19 10:32:31:667 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:896 ]:	Etag poll timer timeout
2023-01-19 10:32:31:667 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:900 ]:	Folders to sync: 1
2023-01-19 10:32:31:667 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:910 ]:	Number of folders that don't use push notifications: 0
2023-01-19 10:32:54:006 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:257 ]	[ OCC::PushNotifications::pingWebSocketServer ]:	Ping websocket server
2023-01-19 10:32:54:211 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:237 ]	[ OCC::PushNotifications::onWebSocketPongReceived ]:	Pong received in time
2023-01-19 10:33:01:676 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:896 ]:	Etag poll timer timeout
2023-01-19 10:33:01:676 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:900 ]:	Folders to sync: 1
2023-01-19 10:33:01:676 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:910 ]:	Number of folders that don't use push notifications: 0
2023-01-19 10:33:24:204 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:257 ]	[ OCC::PushNotifications::pingWebSocketServer ]:	Ping websocket server
2023-01-19 10:33:24:407 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:237 ]	[ OCC::PushNotifications::onWebSocketPongReceived ]:	Pong received in time
2023-01-19 10:33:31:681 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:896 ]:	Etag poll timer timeout
2023-01-19 10:33:31:681 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:900 ]:	Folders to sync: 1
2023-01-19 10:33:31:681 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:910 ]:	Number of folders that don't use push notifications: 0
2023-01-19 10:33:54:403 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:257 ]	[ OCC::PushNotifications::pingWebSocketServer ]:	Ping websocket server
2023-01-19 10:33:54:605 [ debug nextcloud.sync.pushnotifications C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\libsync\pushnotifications.cpp:237 ]	[ OCC::PushNotifications::onWebSocketPongReceived ]:	Pong received in time
2023-01-19 10:34:01:672 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:896 ]:	Etag poll timer timeout
2023-01-19 10:34:01:672 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:900 ]:	Folders to sync: 1
2023-01-19 10:34:01:672 [ info nextcloud.gui.folder.manager C:\Users\sysadmin\AppData\Local\Temp\2\windows-13461\client-building\desktop\src\gui\folderman.cpp:910 ]:	Number of folders that don't use push notifications: 0
```

**Comment by vagner-dias** [comment]
Just to add something I realized in the logs > On my coputer I have no folder such as "C:\Users\sysadmin\", or "client-building", etc..

This seems to be hardcoded.

**Comment by nuokh** [comment]
I've got the same issue despite the file being synced correctly.

- NC version 25.0.3 (docker)
- NC desktop client 3.6.6
- Windows 10 & Chrome

![image](https://user-images.githubusercontent.com/9081831/214827973-5acc508c-004e-49ac-af71-33656755190f.png)

**Comment by allexzander** [comment]
> I've got the same issue despite the file being synced correctly.
> 
> * NC version 25.0.3 (docker)
> * NC desktop client 3.6.6
> * Windows 10 & Chrome
> 
> ![image](https://user-images.githubusercontent.com/9081831/214827973-5acc508c-004e-49ac-af71-33656755190f.png)

Is the folder where "New document.odt" is located, marked for sync in the selective sync dialog? Can you see that file in you local sync folder?

**Comment by nuokh** [comment]
> > I've got the same issue despite the file being synced correctly.
> > 
> > * NC version 25.0.3 (docker)
> > * NC desktop client 3.6.6
> > * Windows 10 & Chrome
> > 
> > ![image](https://user-images.githubusercontent.com/9081831/214827973-5acc508c-004e-49ac-af71-33656755190f.png)
> 
> Is the folder where "New document.odt" is located, marked for sync in the selective sync dialog? Can you see that file in you local sync folder?

Yes.

**Comment by allexzander** [comment]
weird, works on my end with same file and folder names

**Comment by vagner-dias** [comment]
Hi. I managed to fixed my issue creating a new user, transferring the ownership of all files to the new user and resyncing everything on the client.

**Comment by nuokh** [comment]
Could reproduce this issue on my private MacOS home computer.

![grafik](https://user-images.githubusercontent.com/9081831/217268461-6be4b549-af36-4f10-8985-137a6781ca70.png)

Installed the 3.7.1 update for the NC desktop client afterwards, tried again, issue still there.

**Comment by rotdrop** [comment]
I have the same error with 3.7.3 on Ubuntu Jammy.

**Comment by paddif** [comment]
I got the Same issue on Nextcloud 25.0.4 and Desktop client 3.7.3 Windows and MacOS

**Comment by alexb-55** [comment]
I got the Same issue on Nextcloud 25.0.4 and Desktop client 3.7.3 Windows.

**Comment by paddif** [comment]
Issue Seems fixed with Desktop Client 3.7.4

**Comment by nuokh** [comment]
Fixed with 3.7.4 for me, too

**Comment by alexb-55** [comment]
Fixed with 3.7.4 for me, too

**Comment by enoch85** [comment]
This issue seems to be fixed with 3.7.4. Closing. Please reopen if issue persists.
